File: pinyougou-parent/pinyougou-interface/src/main/java/cn/itcast/core/service/AddressService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.address.Address;

import java.util.List;

public interface AddressService {
    List<Address> findListByLoginUser(String name);

    void delete(Long id);

    void setDefault(String name,Long id);

    void update(Address entity);

    void add(Address tbAddress,String name);

    Address findById(Long id);
}

File: pinyougou-parent/pinyougou-web-user/src/main/webapp/js/service/addressService.js
//服务层
app.service('addressService',function($http){
    //读取列表数据绑定到表单中
    this.showAddress=function(){
        return $http.get('../address/findAddressList.do');
    }
    this.delete=function(id){
        return $http.get('../address/delete.do?id='+id);
    }
    this.setDefault=function(id){
        return $http.get('../address/setDefault.do?id='+id);
    }
    this.findById=function(id){
        return $http.get('../address/findById.do?id='+id);
    }
    this.update=function(entity){
        return $http.post('../address/update.do',entity);
    }
    this.add=function(tbAddress){
        return $http.post('../address/add.do',tbAddress);
    }
});